tests/plotting/test_plotting.py

```diff
@@ -34,6 +34,16 @@
 except:
     ffmpeg_failed = True
 
+try:
+    from vtkmodules.vtkCommonCore import vtkVersion
+    vtk_dev = len(str(vtkVersion().GetVTKBuildVersion())) > 2
+except:
+    vtk_dev = False
+
+# These tests fail with mesa opengl on windows
+skip_windows_dev_whl = pytest.mark.skipif(os.name == 'nt' and vtk_dev,
+                                          reason='Test fails on Windows with dev wheels with ')
```

Comment:
Unless there's some trickery with the reasons, the end bit might be superfluous:
```suggestion
                                          reason='Test fails on Windows with VTK dev wheels')
```

Reply:
Quite superfluous.